README.md

```diff
@@ -1,345 +1,451 @@
-# RabbitMQ RPC
-
-[![GoDoc](https://godoc.org/github.com/bombsimon/amqp-rpc?status.svg)](https://godoc.org/github.com/bombsimon/amqp-rpc)
-[![Build Status](https://travis-ci.org/bombsimon/amqp-rpc.svg?branch=master)](https://travis-ci.org/bombsimon/amqp-rpc)
-[![Go Report Card](https://goreportcard.com/badge/github.com/bombsimon/amqp-rpc)](https://goreportcard.com/report/github.com/bombsimon/amqp-rpc)
-[![Maintainability](https://api.codeclimate.com/v1/badges/77ecbf483dc76d4327a5/maintainability)](https://codeclimate.com/github/bombsimon/amqp-rpc/maintainability)
-[![Test Coverage](https://api.codeclimate.com/v1/badges/77ecbf483dc76d4327a5/test_coverage)](https://codeclimate.com/github/bombsimon/amqp-rpc/test_coverage)
-[![golangci](https://golangci.com/badges/github.com/bombsimon/amqp-rpc.svg)](https://golangci.com/r/github.com/bombsimon/amqp-rpc)
+<h1 align="center">
+  <img src="amqprpc.png" alt="AMQP RPC" width="300">
+  <br>
+  RabbitMQ RPC
+  <br>
+</h1>
+
+<h4 align="center">RabbitMQ RPC Framework</h4>
+
+<p align="center">
+  <a href="https://godoc.org/github.com/bombsimon/amqp-rpc">
+    <img src="https://godoc.org/github.com/bombsimon/amqp-rpc?status.svg" alt="GoDoc">
+  </a>
+  <a href="https://travis-ci.org/bombsimon/amqp-rpc">
+    <img src="https://travis-ci.org/bombsimon/amqp-rpc.svg?branch=master">
+  </a>
+  <a href="https://goreportcard.com/report/github.com/bombsimon/amqp-rpc">
+    <img src="https://goreportcard.com/badge/github.com/bombsimon/amqp-rpc">
+  </a>
+  <a href="https://codeclimate.com/github/bombsimon/amqp-rpc/maintainability">
+    <img src="https://api.codeclimate.com/v1/badges/77ecbf483dc76d4327a5/maintainability">
+  </a>
+  <a href="https://codeclimate.com/github/bombsimon/amqp-rpc/test_coverage">
+    <img src="https://api.codeclimate.com/v1/badges/77ecbf483dc76d4327a5/test_coverage">
+  </a>
+  <a href="https://golangci.com/r/github.com/bombsimon/amqp-rpc">
+    <img src="https://golangci.com/badges/github.com/bombsimon/amqp-rpc.svg">
+  </a>
+</p>
 
 ## Description
 
-This is a framework to use [RabbitMQ](https://www.rabbitmq.com) with
-[Go amqp](https://github.com/streadway/amqp) as RPC client/server setup. The
-purpose of this framework is to implement a fully functional message queue setup
-where a user can just plug in handlers on the server(s) and use the client to
-communicate between them. This is suitable for many micro service architectures.
+This is a framework to use [RabbitMQ] as a client/server RPC setup togheter with
+the [Go amqp] implementation. The framework can manage a fully funcitonal
+message queue setup with reconnects, disconnects, graceful shutdown and other
+stability mechanisms. By providing this RabbitMQ can be used as transport and
+service discovery to quickly get up and running with a micro service
+architecture.
+
+Since the framework handles the majority of the interface with RabbitMQ and the
```

Comment:
Since the framework handles the majority of the communication with RabbitMQ?

Reply:
Feel free to propose enhancements or other ways of writing!

Reply:
That was my proposal :)

Reply:
Oops!